tools/depends/target/smctemp/SMCTEMP-VERSION

```diff
@@ -0,0 +1,5 @@
+LIBNAME=smctemp
+VERSION=0.2.1
+ARCHIVE=$(LIBNAME)-$(VERSION).tar.gz
+SHA512=8671836ed3f16122ffc84a1e91b463c8405526b1500fa9f5816a9f9eff1fd598c86958e894d2b8b25ae798da57b536766729b28dcc6b7c69fe2dc3c818f18290   
```

Comment:
You've got spaces at the end of the SHA512 hash. you'll need to remove them.

Reply:
Thanks fuzzard, for some reason I could not check the error on the phone screen.
Are you happy with the current state of the PR? I know you suggested to change to upstream makefile but since this is so simple (library is only a couple of files) imho it doesn't really justify.

Reply:
Fine by me.